dependencies/graknlabs/artifacts.bzl

```diff
@@ -29,3 +29,13 @@ def graknlabs_grakn_core_artifacts():
         commit_source = deployment["artifact.snapshot"],
         commit = "b9d1cb52a7e261995d2f3650bbcf2626b4f62c3e",
     )
+
+def graknlabs_grakn_cluster_artifacts():
```

Comment:
Now that we have both Core and Cluster I think we need to seriously consider if there's a better alternative than downloading 400MB of Grakn only to throw away 2/3rds of it.

I wouldn't know how to achieve this but we should definitely raise an issue I think.

Reply:
Why would we throw away 2/3 of it? From that point onwards we are all expected to test both Grakn Core and Cluster right?

In any case, the solution is the fact that we're going to move cross-product testing to the `test` repo. The repo will have to be optimised to not have to always pull huge artifacts.

Reply:
Because if you're running on Linux, the Windows and Mac artifacts are of no use to you.

Reply:
Ah yeah I see what you mean.

Okay, let's keep it this way and I'll create an issue for the `test` repo with regards to minimising download size in `grakn` (since we don't yet have the `test` repo).

FYI, the approach that we thought to try is to use multiple `WORKSPACE` file inside the `test` repo. We can have a different Bazel workspace per platform and also per product integration. The permutation will be high and we'll need to have an additional tooling to manage it, but I don't see any other way.